fyndiq/django-gstorage
# Config skeleton

### Purpose

This PR specifies an app config as recommended here - https://docs.djangoproject.com/en/1.10/ref/applications/. The app throws an error if any of the required configurations are missing. Not all the configurations are listed. Instead, we focus on establishing the initial skeleton for the app.

### How to test
1. `tox`
1. Add 'gstorage' as a required app in an existing app and check errors when we do, for example, `python scripts/manage.py shell`
